src/InvocationModel.ts

```diff
@@ -62,7 +63,13 @@ export class InvocationModel implements coreTypes.InvocationModel {
                 const bindingName = nonNullProp(binding, 'name');
                 let input: unknown = fromRpcTypedData(binding.data);
 
-                const bindingType = nonNullProp(this.#bindings, bindingName).type;
+                const rpcBinding = this.#bindings[bindingName];
+                if (!rpcBinding) {
+                    throw new AzFuncSystemError(
+                        `Failed to find binding "${bindingName}" in bindings "${Object.keys(this.#bindings).join()}".`
```

Comment:
Should we consider `.join(", ")` instead of `.join()` so that there is some spacing in the list of keys? Right now, we log the bindings without any separation: `"httpTrigger1,$return".'`